cluster-up/cluster/kind-k8s-sriov-1.17.0/provider.sh

```diff
@@ -31,5 +33,32 @@ function up() {
 EOF
 
     setup_kind
+
+    # remove the rancher.io kind default storageClass
+    _kubectl delete sc standard
+
+    nodes=$(_kubectl get nodes -o=custom-columns=:.metadata.name | awk NF)
+    for node in $nodes; do
+        # Create local-volume directories, which, on other providers, are pre-provisioned.
+        # For more info, check https://github.com/kubevirt/kubevirtci/blob/master/cluster-provision/STORAGE.md
+        for i in {1..10}; do
+            mount_disk $node $i
+        done
+        $DOCKER exec $node bash -c "chmod -R 777 /var/local/kubevirt-storage/local-volume"
```

Comment:
do we have a need / test on kubevirt for this ?

Reply:
Its something that you want to have in dev environment so you could create PVC's locally 
it also enable you to create VM's with persistent storage

Reply:
can you add a test for it in kubevirt please? unless we have one,
else we are adding a unchecked code.

Reply:
On this line https://github.com/kubevirt/kubevirtci/pull/470/files#diff-fcaa29a18b4077ecaa46dca6e69b637432306c3d2f38d3def35ff59ecc33dbbfR52
as part of this manifest we create local-volume-provisioner daemon set.
We check for all pods readiness all over cluster-up if something will go wrong with those pods it will be reported.

Reply:
i am worried from this change, 
especially when we dont use it
i saw couple times locally issues of local-volume provisioner pods that arent started well and so on

Reply:
I will remove since its not widely use as you said, 
if we will need it for development we will bring it back